index.d.ts

```diff
@@ -120,3 +127,37 @@ console.log(error);
 ```
 */
 export function deserializeError(errorObject: ErrorObject | unknown, options?: Options): Error;
+
+/**
+Predicate to determine whether a value looks like an error, even if it's not an instance of `Error`. It must have at least the `name`, `message`, and `stack` properties.
+
+@example
+```js
```

Comment:
```suggestion
```
```

Reply:
The rest of the examples use \`\`\` too. I dropped `js` though

Reply:
> If you want to drop ``` from all of them it can be done in a separate commit I suppose

👍